tests/draft2019-09/optional/format/ipv6.json

```diff
@@ -22,6 +22,46 @@
                 "description": "an IPv6 address containing illegal characters",
                 "data": "::laptop",
                 "valid": false
+            },
+            {
+                "description": "no digits is valid",
+                "data": "::",
+                "valid": true
+            },
+            {
+                "description": "leading colons is valid",
+                "data": "::1",
```

Comment:
This exactly duplicates the first test (`a valid IPv6 address`). Can you either remove that test or replace one of the two with a different enough value?

Reply:
oops, fixing.